components/brave_ads/core/internal/common/strings/string_conversions_util.cc

```diff
@@ -18,4 +22,30 @@ std::string BoolToString(const bool value) {
   return value ? kTrue : kFalse;
 }
 
+std::vector<float> ConvertStringToVector(const std::string& string,
+                                         const std::string& delimiter) {
+  const std::vector<std::string> vector_string = base::SplitString(
```

Comment:
this is not a string, so `vector_string` could be confusing. How about `components`?